### DISCUSSION TITLE
/articles/sharepoint-embedded-frequently-asked-questions/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#274] [Comments]
In this article, I answer some of the frequently asked questions I’ve received about a new service from Microsoft 365 - SharePoint Embedded.
https://www.andrewconnell.com/articles/sharepoint-embedded-frequently-asked-questions/

**Comment by daneel3001** [comment]
Great content!

I went through your guide: https://www.voitanos.io/blog/sharepoint-embedded-create-apps/. 

However it's not clear to me how and where regular users who needs access to content via SPE are registered.
My context is a B2B SaaS application where I want to leverage SPE. Each tenant of the application being strictly isolated from each other, that includes both files and users and their permissions.

I have my own Identity Platform which may not be Entra ID and I would like to enable my SaaS application users to have access to documents. 

I understand the concept of File Storage Containers and Container Types, however it's not clear to me whether the File Storage Container will as well contain the users having some kind of access to it. 
To reformulate the question, does File Storage Container comes with a scoped set of users ?

**Reply by andrewconnell** [reply]
Not sure I understand this part of your question:

> To reformulate the question, does File Storage Container come with a scoped set of users?

But, users live in Entra ID, but those can be guest users or federated from your third-party identity provider, just like when you add someone as a guest/external user to SharePoint Online or Microsoft Teams.

SharePoint Online (and all of Microsoft 365) uses Entra ID as its identity provider... Entra ID has capabilities to support third-party identity providers.

Containers don't have their own user store... you're simply granting a permission role (owner/manager/writer/reader) to an existing user on a Container.

> My context is a B2B SaaS application where I want to leverage SPE. Each application tenant is strictly isolated from each other, including files and users and their permissions.

So, each tenant has its own Entra ID where users are stored. The content in your SPE app lives in Containers that reside in the SPE partition in each of those Microsoft 365 tenants... so all the content & users & permissions are stored within your customer's tenant.

The only thing you as the SaaS retain is the application (code) that knows how to read/write content to your customer's tenants.

**Comment by lawrenceAtWork** [comment]
hello, 

Just wondering if one can use termstore terms in the column metadata? 

tks,
Lawrence

**Reply by andrewconnell** [reply]
Not at this time.
